.changeset/sour-frogs-shout.md

```diff
@@ -0,0 +1,25 @@
+---
+'astro': patch
+---
+
+Added support for optimizing remote images to `astro:assets`. This comes with two new parameters to specify which domains and host patterns are authorized for remote images.
```

Comment:
```suggestion
Added support for optimizing remote images from authorized sources when using `astro:assets`. This comes with two new parameters to specify which domains (`image.domains`) and host patterns (`image.remotePatterns) are authorized for remote images.
```
I think adding "from authorized sources" right in the first sentence is helpful for "skimmers", because that will immediately make them realize that they need to somehow *do* something re: specifying authorization, vs just "Yay, remote images now!"